cpg-core/src/main/java/de/fraunhofer/aisec/cpg/helpers/NameConverter.kt

```diff
@@ -41,12 +41,14 @@ class NameConverter : CompositeAttributeConverter<Name> {
 
     companion object {
         const val FIELD_FULL_NAME = "fullName"
+        const val FIELD_NAME = "name"
```

Comment:
Can we have a comment here saying that name is an alias for fullName? Or where is the best place to mention that in code, the member where the name is stored?

Reply:
I added some comments to the persistence function